README.md

```diff
@@ -1 +1,17 @@
-#pnc-js-libs
\ No newline at end of file
+#pnc-js-libs
+
+library at the moment mainly process pnc notifications
+- pnc-dto-types - dto types generated from swagger json file
```

Comment:
I would provide more detail information about swagger json file - maybe mentioning `pnc-dto-types/generate-openapi-spec.sh` could be useful to have better understanding